scripts/index.md

```diff
@@ -0,0 +1,41 @@
+# Available scripts index
+
+## Scripts and processes endpoints
+This index contains a list of all the provided scripts in DSpace REST API contract, to learn more about the scripts and processes endpoints, they are available here: 
```

Comment:
could be better to end the phrase with "check the following documentation:" instead that "they are available here:" ?